### ISSUE TITLE
CameraController -> startByteStream missing Uint8List 

### ISSUE THREAD

**Issue by rai2270** [bparrishMines/mlkit_demo#4]
Hi, 
I'm trying to find a way to call CameraController ->  startByteStream with Uint8List.

In your branch I can find : startByteStream(CameraImage image) , and I saw that you use it like that in the demo. But Firebase changes show FirebaseVisionImage.fromBytes( Uint8List bytes, ... )  receive Uint8List.

I assume Firebase need to stay with byte stream , so we need a way to convert CameraImage to Uint8List.

Can you share the way you make that conversion?

Thanks.

**Comment by bparrishMines** [comment]
Yes, I just updated this demo a little while ago. Once the PRs go through I will also update the `firebase_ml_vision` example with a live camera view example.

**Comment by rai2270** [comment]
thanks. working :)  great work
